front/app/containers/Admin/projects/project/events/edit.tsx

```diff
@@ -178,7 +178,9 @@ const AdminProjectEventEdit = () => {
   useEffect(() => {
     if (eventAttrs.address_1 !== event?.data.attributes.address_1) {
       const delayDebounceFn = setTimeout(async () => {
-        const point = await geocode(eventAttrs.address_1);
+        const point = eventAttrs.address_1
+          ? await geocode(eventAttrs.address_1)
+          : null;
         setGeocodedPoint(point);
```

Comment:
@jinjagit This is a bit of a weird fix, but it's the only way this works correctly + consistently for now.

Reply:
Thanks!